scripts/core/platform.yml

```diff
@@ -242,3 +242,40 @@ etors:
     - name: HIP
       value: "4"
       desc: "The backend is HIP"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Extension name maximum length."
+name: $X_MAX_EXTENSION_NAME_LENGTH
+value: "256"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties."
+name: $x_extension_properties_t
+base: $x_base_properties_t
```

Comment:
I don't think this is the expected usage of `_properties_t` which are intended for creating objects with `pNext` chains, not querying info about things.